sandcastle/api.py

```diff
@@ -74,6 +74,8 @@
 logger = logging.getLogger(__name__)
 
 WEBSOCKET_CALL_TIMEOUT = 30.0  # seconds
+# try to initiate WS Client 5 times
+MAGIC_KONSTANT = 5
```

Comment:
If you had chosen a better name, you wouldn't need the comment...;-)

Reply:
there is no better name for that constant :P